src/components/TabPresale/TabPresale.tsx

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable react/no-array-index-key */
+import React, { Children, isValidElement } from "react";
+import styled from "styled-components";
+import { space } from "styled-system";
+import Box from "../Box/Box";
+import { TabPresaleProps } from "./types";
+
+const StyledTabPresale = styled(Box)`
+  align-items: center;
+  display: flex;
+  list-style-type: none;
+  ${space};
+`;
+
+const ChildTab = styled.li<{ isActive: boolean; index: number }>`
+  min-width: max-content;
+  cursor: pointer;
+  margin-left: ${({ index }) => (index === 0 ? "0" : "12px")};
+  margin-right: 12px;
+  padding-bottom: 5px;
+  border-bottom: 5px solid
+    ${({ theme, isActive }) =>
+      isActive ? ` ${theme.colors.primary}` : "transparent"};
+  > div {
+    color: ${({ theme, isActive }) => isActive && theme.colors.primary};
+    font-weight: ${({ isActive }) => isActive && "700"};
+  }
+  ${({ theme }) => theme.mediaQueries.md} {
+    margin-left: ${({ index }) => (index === 0 ? "0" : "16px")};
+    margin-right: 16px;
+  }
+`;
+
+const TabPresale: React.FC<TabPresaleProps> = ({
```

Comment:
i think we shouldn't put the word presale here, as it can be use anywhere, not just presale feature